concrete/src/Sharing/SocialNetwork/ServiceList.php

```diff
@@ -28,7 +28,18 @@ protected static function getServices()
         );
 
         if ($additionalSocialNetworks = \Config::get('concrete.social.additional_services')) {
-            $services = $additionalSocialNetworks + $services;
+            $keyArray = [];
+            $additionalKeyArray = [];
+
+            foreach($services as $service) {
+                $keyArray[$service[0]] = $service;
+            }
+
+            foreach($additionalSocialNetworks as $service) {
```

Comment:
Same here, needs PSR-2 spacing.
It'd also be good to add some comments that describe what this is doing and why.

Reply:
Do you mean comments in the code, or commits in the commit?

Reply:
I think in the PHP code